dev/replace_aws_with_gcp_env_var.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# This script is to update all AWS (or default cloud provider) enviroment variables with GCP values
+# It's used to run tests against GCP
+
+# Iterate over all environment variables
+for var in $(env | cut -d= -f1); do
```

Comment:
instead of doing this, maybe we have a "router" level CI var that specifies which env to test in? and the value of the variable (defaulting to aws lve for example) determines which CI secrets it picks up?

so the var list would look like (for example)
- AWS_DEV_AUDIT_DOMAIN
- AWS_DEV_AUDIT_TOKEN
- GCP_DEV_AUDIT_DOMAIN
- GCP_DEV_AUDIT_TOKEN
- TEST_SELECTOR

if TEST_SELECTOR == "GCP_DEV" it'll use `GCP_DEV_AUDIT_DOMAIN` and `GCP_DEV_AUDIT_TOKEN` and we just update our selector value in each test run

Reply:
Right now we can select test environment by service due to some of them are in develop. It's true that all those services that are in main should be tested against live, but in that case we should split cicd testing in both cases main branch and develop branches. We are still discussing about how to properly implement this with @nmader-pangea and @kenany